src/Components/Web.JS/src/Rendering/Renderer.ts

```diff
@@ -24,7 +24,7 @@ export function attachRootComponentToLogicalElement(browserRendererId: number, l
   browserRenderer.attachRootComponentToLogicalElement(componentId, logicalElement, appendContent);
 }
 
-export function attachRootComponentToElement(elementSelector: string, componentId: number, browserRendererId?: number): void {
+export function attachRootComponentToElement(elementSelector: string, componentId: number, browserRendererId: number): void {
```

Comment:
Since browserRenderId is now required could the fallback to 0 in line 47 be removed?
`browserRendererId || 0`

Reply:
Yeah, it could be. I can remove it the next time I touch this area, but feel free to open a PR if you feel inclined :)